server.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const express = require('express');
+
+// Constants
+const PORT = 8080;
+
+// App
+const app = express();
+app.get('/', function (req, res) {
+   res.send('Hello world\n');
```

Comment:
Not sure what the scope of the ticket is, this certainly works but there's not other functionality.